lib/webpacker/manifest.rb

```diff
@@ -105,11 +105,14 @@ def manifest_type(pack_type)
     def missing_file_from_manifest_error(bundle_name)
       <<-MSG
 Webpacker can't find #{bundle_name} in #{config.public_manifest_path}. Possible causes:
-1. You want to set webpacker.yml value of compile to true for your environment
-   unless you are using the `webpack -w` or the webpack-dev-server.
-2. webpack has not yet re-run to reflect updates.
-3. You have misconfigured Webpacker's config/webpacker.yml file.
-4. Your webpack configuration is not creating a manifest.
+1. You forgot to install node packages (try either `npm install` or `yarn install`)
+2. Your app has code with a non-standard extension (like a `.jsx` file) but the extension is not in the `extensions` config in `config/webpacker.yml`
+3. You have set compile: false (see `config/webpacker.yml`) for this environment
+   (unless you are using the `webpack -w` or the webpack-dev-server, in which case maybe you aren't running the dev server in the background?)
+4. Webpacker has not yet re-run to reflect updates.
```

Comment:
`bin/webpack` not Webpacker?

Reply:
you're right it was `webpack` before my change on that line was idle and not intentional